assets/src/edit-story/components/library/text/fontPreview.js

```diff
@@ -45,18 +45,20 @@ const Text = styled.span`
   background: none;
   font-size: ${({ fontSize }) => fontSize * PREVIEW_EM_SCALE}px;
   font-weight: ${({ fontWeight }) => fontWeight};
-  font-family: ${({ fontFamily }) => fontFamily};
+  font-family: ${({ font }) => font.family};
   color: ${({ theme }) => theme.colors.fg.v1};
 `;
 
-function FontPreview({ title, fontFamily, ...fontProps }) {
+function FontPreview({ title, ...fontProps }) {
   const {
     actions: { maybeEnqueueFontStyle },
   } = useFont();
 
+  const { font } = fontProps;
```

Comment:
Why not just expand it up to as `{ title, font, ...fontProps}`?

Reply:
`font.family` needs to be part of `fontProps` too

Reply:
This would mean that `fontProps` wouldn't include `font` for the `Text` below though. Perhaps better to split it up anyway.

Reply:
Refactored this a bit.